source/includes/fact-save-pipeline.rst

```diff
@@ -1,24 +1,21 @@
-You can save a pipeline so that you can find it later. If you
-load a saved pipeline, you can change it without
-changing the original saved copy. You can also create a :ref:`view
-<view-details>` from your pipeline results.
-
-To save your pipeline:
-
 .. procedure::
    :style: normal
 
-   .. step:: Click the save dropdown button
+   .. step:: Click the save dropdown button.
 
-      Click the :guilabel:`Save` dropdown button at the top of the pipeline
-      builder. Click on ``Save pipeline as...``
+      In the aggregation pipeline pane, click the :guilabel:`Save` drop-down 
+      menu and select ``Save pipeline as...``.
```

Comment:
[question] The image doesn't include the phrase "Save pipeline as..." -- does the image need to be updated, or does the text?

Reply:
Updating the text!